example/android/app/build.gradle

```diff
@@ -117,8 +117,7 @@ android {
         versionCode 1
         versionName "1.0"
         testBuildType System.getProperty('testBuildType', 'debug')
-        missingDimensionStrategy "minReactNative", "minReactNative46"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Any particular reason to move to `androidx`? I haven't seen anything in Detox's migration guide that mention it. I've seen (and had) a bad experience with mixing support libs with `androidx` in one project, so ideally we'd love to avoid that conflict.

Reply:
When you migrate from 10.x to 11.x androidx is necessary for detox